buildout.cfg

```diff
@@ -62,6 +62,7 @@ eggs =
     pytest-cov
     pytest-watch
     guillotina [test]
+    cookiecutter
```

Comment:
not sure, if setup.py should depend on it because setup.py also installs the console_scripts...